rc/filetype/janet.kak

```diff
@@ -4,12 +4,12 @@
 # Detection
 # ‾‾‾‾‾‾‾‾‾
 
-hook global BufCreate .*[.](janet|jdn) %{
+hook -group janet global BufCreate .*[.](janet|jdn) %{
     set-option buffer filetype janet
 }
 
 
-hook global WinSetOption filetype=janet %{
+hook -group janet global WinSetOption filetype=janet %{
```

Comment:
The traditional way of excluding bundled kak files is to create your own [autoload directory](https://github.com/mawww/kakoune#configuration-autoloading).

Your solution seems fine too though I don't immediately see why a third party janet.kak would want to suppress these benign hooks.

Reply:
I want third party janet.kak moduels to start with a clean plate. The simpler, the better.